.github/workflows/python-app.yml

```diff
@@ -21,64 +21,52 @@ jobs:
 
     - uses: actions/checkout@v4
 
-    - name: Vertex AI authentication
-      uses: 'google-github-actions/auth@v2'
-      with:
-        credentials_json: '${{ secrets.VERTEX_AI_CREDENTIALS_JSON }}'
-
     - name: Write secrets
       env:
         SECRETS_CONFIG: |
           [anthropic]
-          api_key = "${{ secrets.ANTHROPIC_API_KEY }}"
+          api_key = "fake"
 
           [together]
-          api_key = "${{ secrets.TOGETHER_API_KEY }}"
+          api_key = "fake"
 
           [openai]
-          api_key = "${{ secrets.OPENAI_API_KEY }}"
+          api_key = "fake"
 
           [hugging_face]
-          token = "${{ secrets.HUGGING_FACE_TOKEN }}"
+          token = "fake"
 
           [google_ai]
-          api_key = "${{ secrets.GOOGLE_AI_API_KEY }}"
+          api_key = "fake"
 
           [mistralai]
-          api_key = "${{ secrets.MISTRAL_AI_API_KEY }}"
+          api_key = "fake"
 
           [vertexai]
-          project_id = "${{ secrets.GOOGLE_PROJECT_ID }}"
+          project_id = "fake"
           region = "us-central1"
 
           [azure_phi_3_5_mini_endpoint]
-          api_key = "${{ secrets.AZURE_PHI_3_5_MINI_API_KEY }}"
+          api_key = "fake"
 
           [azure_phi_3_5_moe_endpoint]
-          api_key = "${{ secrets.AZURE_PHI_3_5_MOE_API_KEY }}"
-
-          [modellab_files]
-          token = "${{ secrets.MODELLAB_FILE_DOWNLOAD_TOKEN }}"
+          api_key = "fake"
 
           [nvidia-nim-api]
-          api_key = "${{ secrets.NVIDIA_API_KEY }}"
+          api_key = "fake"
 
           [demo]
           api_key="12345"
 
+          [modellab_files]
```

Comment:
What's this token have access to?

Reply:
In that case, i'd rather remove this unless there's a strong need. The official prompts shouldn't be used by anybody downloading this from PyPi, and any leak of that token would be very expensive, as the best response would be to create all new leaked prompts.